packages/loopring_v3/contracts/core/impl/DefaultDepositContract.sol

```diff
@@ -52,6 +52,7 @@ contract DefaultDepositContract is IDepositContract, Claimable
         address _exchange
         )
         external
+        onlyOwner
```

Comment:
The contract is used behind a proxy, so when calling `initialize` there is no owner yet.